plugins/woocommerce-admin/client/wp-admin-scripts/product-tracking/shared.ts

```diff
@@ -418,10 +418,69 @@ export const initProductScreenTracks = () => {
 				'.woocommerce_attribute'
 			).length;
 			if ( newAttributesCount > attributesCount ) {
+				const global_attributes = [
+					...document.querySelectorAll( '.pa_glbattr' ),
+				].map( ( globalAttr ) => {
+					const terms =
+						(
+							globalAttr.querySelector(
+								'.attribute_values'
+							) as HTMLSelectElement
+						 ).selectedOptions?.length ?? 0;
+
+					return {
+						visibleOnTheProductPage: (
+							globalAttr.querySelector(
+								'[name="attribute_visibility[1]"'
+							) as HTMLInputElement
+						 )?.checked
+							? 'yes'
+							: 'no',
+						usedForVariations: (
+							globalAttr.querySelector(
+								'[name="attribute_variation[1]"'
+							) as HTMLInputElement
+						 )?.checked
+							? 'yes'
+							: 'no',
+						terms,
+					};
+				} );
+				const local_attributes = [
+					...document.querySelectorAll(
+						'.woocommerce_attribute:not(.pa_glbattr)'
+					),
+				].map( ( attr ) => {
+					const terms =
+						(
+							attr.querySelector(
+								"[name='attribute_values[0]']"
+							) as HTMLTextAreaElement
+						 )?.value.split( '|' ).length ?? 0;
+					return {
+						visibleOnTheProductPage: (
+							attr.querySelector(
+								'[name="attribute_visibility[1]"'
+							) as HTMLInputElement
+						 )?.checked
+							? 'yes'
+							: 'no',
+						usedForVariations: (
+							attr.querySelector(
+								'[name="attribute_variation[1]"'
+							) as HTMLInputElement
+						 )?.checked
+							? 'yes'
+							: 'no',
+						terms,
+					};
+				} );
 				recordEvent( 'product_attributes_add', {
 					page: 'product',
```

Comment:
Afaiu, the event modified here relates to local attributes (added from the product tab), while the one [in this file](https://github.com/woocommerce/woocommerce/blob/9d7e503519f96ca6229445f3e1bf1e59a3d956fa/plugins/woocommerce-admin/client/wp-admin-scripts/attributes-tracking/index.js#L14) to the global ones (added via `wp-admin/edit.php?post_type=product&page=product_attributes` ). Do you agree?

Reply:
Yes. However, I can save information about global attributes added to the product here.

Reply:
True. At the same time, we can opt to keep the events data "on point" (relevant to its purpose: here, adding a new element) and use SQL to shape/extract whatever information is needed at time of an analysis.